public/js/test/mochitest/head.js

```diff
@@ -204,3 +221,27 @@ function isVisibleWithin(outerEl, innerEl) {
   return innerRect.top > outerRect.top &&
     innerRect.bottom < outerRect.bottom;
 }
+
+
+const selectors = {
+  callStackArrow: ".call-stack-pane .arrow",
+  frame: (index) => `.frames ul li:nth-child(${index})`
+}
+
+function findElement(dbg, elementName, ...args) {
+  let selector = selectors[elementName];
+  if (typeof selector == "function") {
+    selector = selector(...args)
+  }
+
+  return dbg.win.document.querySelector(selector);
+}
+
+function toggleCallStack(dbg) {
+  return findElement(dbg, "callStackArrow").click()
+}
+
+Services.prefs.setBoolPref("devtools.debugger.new-debugger-frontend", true);
+registerCleanupFunction(() => {
+  Services.prefs.setBoolPref("devtools.debugger.new-debugger-frontend", false);
```

Comment:
I think you've duplicated this. It's already at the top of `head.js`, so we don't need this here.